Build version 24.10.2

beenje opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

This recipe hasn't been updated in a while. I need it to be compiled with python 3.13 for another feedstock.
Python 3.13 support as added in 24.10.1.

- Added myself as maintainer
- Updated the requirements based on version 24.10.2 [pyproject.toml](https://github.com/gevent/gevent/blob/24.10.2/pyproject.toml) and [setup.py](https://github.com/gevent/gevent/blob/24.10.2/setup.py)
- Removed the patches - they were already not used anymore as this was to build the vendored dependencies and `GEVENTSETUP_EMBED_XXX` were set to 0
- Use `GEVENTSETUP_EMBED=0` instead of individual variables
- Run some tests (I tried running all but some failed randomly in CI - running just a few is enough)

beenje commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

beenje commented:
@conda-forge-admin, please rerender

beenje commented:
@conda-forge-admin, please rerender

beenje commented:
@conda-forge/core, this feedstock hasn't been updated in a while.
I added myself as maintainer and updated the recipe. Could someone please merge? Thanks!

beenje commented:
> We usually give maintainers ~1 week before merging. Please ping us again when the time passed.

It has been a week :-) @conda-forge/core, gentle ping.
